src/app/balance-game/[roomId]/page.tsx

```diff
@@ -0,0 +1,57 @@
+import Chatting from '@/components/Chatting';
+import { CHAT_DUMMY, MY_NAME, USER_DUMMY, GAME_INFO } from './DUMMY';
+import UserInfoCard from '@/components/UserInfoCard';
+import { GameListCard } from '@/components/GameListCard';
+import { Button } from '@/components/Button';
+import { Loader, Link, CheckCheck } from 'lucide-react';
+
+const WaitingRoom = () => {
+  const readyCount = USER_DUMMY.filter(({ isReady }) => isReady).length;
```

Comment:
P4; 필터도 좋지만 reduce가 조금 더 나을 것 같습니다. filter는 조건을 만족하는 요소를 새로운 배열에 복사한 뒤, 해당 배열의 길이를 계산하기 위해 또 다른 연산을 추가로 수행하지만, reduce로 처리할 경우 한번에 처리할 수 있습니다!
```suggestion
  const readyCount = USER_DUMMY.reduce((count, { isReady }) => count + (isReady ? 1 : 0), 0);
```

Reply:
오호 불필요한 연산을 줄일 수 있어서 좋네요! reduce를 잘 활용해보도록 신경 써야겠네요 알려주셔서 감사합니다~~